generator/uscore/uscore_generator.rb

```diff
@@ -716,7 +716,7 @@ def structure_to_string(struct)
       def create_resource_profile_test(sequence)
         test_key = :validate_resources
         test = {
-          tests_that: "#{sequence[:resource]} resources returned from previous search conform to the #{sequence[:profile_name]}.",
+          tests_that: "#{sequence[:resource]} resources returned conform to the #{sequence[:profile_name]}.",
```

Comment:
I think we should run this wording change by @arscan.

Reply:
I think I'm ok with it.  For these, try to put yourself in the position of a developer that is looking at these tests for the first time.  If they fail the test, will they ask "what resources returned where?"  I would like to train people to be reading the test details if it is hard to be succinct here, so maybe that's the answer (we beef up the details section).  Or maybe we can just state "...resources received during previous tests...".

Reply:
Now that we are really getting into the wording here, this isn't quite accurate.  Sometimes we only look at SOME of the resources, because not all Observation resource types need to conform to the BodyWeight profile, for example.  Should we instead say "Applicable #{sequence[:resource]} resources received during previous tests"?

Reply:
I like that wording, still succinct but more descriptive, I'll make the update

Reply:
I was looking at the other tests, and the downside of rewording it too much is that then the wording may start misaligning with the language in the other tests, so we get into the situation where we are renaming all tests.  I think maybe just going with "returned during previous tests" is good and we'll leave it at that.  Thansk!